superset-frontend/src/chart/Chart.jsx

```diff
@@ -256,6 +256,13 @@ class Chart extends React.PureComponent {
 
     const isLoading = chartStatus === 'loading';
     const isFaded = refreshOverlayVisible && !errorMessage;
+    // redefine the formData because the x & y axis title margin of formData is string type in case of custom value. And so these should be always number.
+    /* eslint radix: ["error", "as-needed"] */
+    const formData = {
+      ...this.props.formData,
+      x_axis_title_margin: parseInt(this.props.formData.x_axis_title_margin),
```

Comment:
I think Chart.jsx shouldn't be aware of such plugin specific controls. I'd suggest parsing those values in `plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts` (and wherever else they're used, I think Echarts Boxplot and MixedTimeseries).
You could create a helper function in `plugins/plugin-chart-echarts/src/utils` that would return a parsed number or 0 if NaN